dist/LICENSE

```diff
@@ -216,6 +216,8 @@ The following components are provided under the Apache License. See project link
 The text of each license is the standard Apache 2.0 license.
     client_golang (prometheus) v1.9.0: https://github.com/prometheus/client_golang Apache 2.0
     grpc-go (grpc) v1.36.1: https://github.com/grpc/grpc-go Apache 2.0
+    prometheus (prometheus) v1.8.2: http://github.com/prometheus/prometheus Apache 2.0
+    go-yaml (yaml) v2: https://github.com/go-yaml/yaml Apache 2.0
```

Comment:
Since https://github.com/apache/skywalking-satellite/pull/49 is also going to use yaml and it uses v3, let's use v3 for this as well, newer version should be more preferable, right?